Sources/PaymentButtons/PayPalButton.swift

```diff
@@ -8,6 +8,9 @@ public final class PayPalButton: PaymentButton {
     public enum Color: String {
         case gold
         case white
+        case black
+        case silver
+        case blue
```

Comment:
Could you attach the PR that has the changes we are reverting? It would help a lot for the review, I think here we need to revert the changes to the default color parameter on _init's._

Reply:
Here's the relevant PR where colors were removed from `next-major-version` https://github.com/paypal/paypal-ios/pull/243

Reply:
The inits should be updated with the original colors, but just check against the original to make sure